PRODUCTIONISATION.md

```diff
@@ -107,17 +106,20 @@ To be applied if infrastructure has been orchestrated and provisioned specifical
 * [ ] 8.1 Sub-organisation created under parent organisation for each customer
 * [ ] 8.2 Monitoring agent be installed on all production infrastructure
 * [ ] 8.3 Monitoring agent could be installed on non-production infrastructure if necessary
-* [ ] 8.4 Monitoring agent hostname configured according to proposed [RFC 008 - Server naming convention][rfc_008]
+* [ ] 8.4 Monitoring agent hostname configured according to proposed [RFC 008 - Server naming convention][rfc_008] (_this doesn't apply to serverless_)
 * 8.5 Monitors configured in infrastructure monitoring service for (warn/alert):
   * [ ] 8.5.1 Inode usage (80% / 90%)
   * [ ] 8.5.2 CPU load (75% / 85% average over 2 minutes)
   * [ ] 8.5.3 Disk usage (80% / 90%)
-  * [ ] 8.5.4 Memory usage (80% / 90% average over 2 minutes)
+  * [ ] 8.5.4 Memory usage (60% / 70% average over 2 minutes)
   * [ ] 8.5.5 Host stopped reporting
   * [ ] 8.5.6 NTP sync
 * 8.6 Monitors configured to alert via:
   * [ ] 8.6.1 Slack channel
   * [ ] 8.6.2 Team email address
+* 8.7 Monitoring agent configured to monitor
```

Comment:
Think this should be under the responsibility of an APM rather than at infrastructure level to better separate the concerns.

Reply:
That is a good idea, thank you. :)